### DISCUSSION TITLE
TTS and STT enabled chatbots.

### DISCUSSION THREAD

**Discussion by atilla-the-hun** [FlowiseAI/Flowise#2712] [Q&A]
This might be a silly question, but does Flowise have this option? TTS (Text-to-speech) and STT (Speech-to-text) is all the rage with deployed chatbots these days. Does Flowise have this feature? Have I missed it somewhere or is currently in development? That would really add a great feature to this great open-source project.

**Comment by kilmarnock** [comment]
Here Mr. van Zyl shows the STT feature:
https://youtu.be/Sf0e9kD5NPE?t=42
